Fieldset/styles.scss

```diff
@@ -2,7 +2,9 @@
 @import '../css/mixins/index';
 
 .fieldset {
+  background-color: white;
```

Comment:
The problem is when how they want this to render. If you look at the design they want the gray to go above the corners of the address panel.